app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+all_variables = [lucky_number, pi, one_is_a_prime_number, name,
```

Comment:
No calculations required, just simply pass the variables in the correct list in dict